tools/mo/openvino/tools/mo/utils/get_ov_update_message.py

```diff
@@ -14,3 +14,13 @@ def get_ov_update_message():
     link = 'https://software.intel.com/content/www/us/en/develop/tools/openvino-toolkit/download.html?cid=other&source=prod&campid=ww_2022_bu_IOTG_OpenVINO-2022-1&content=upg_all&medium=organic'
 
     return msg_fmt.format(link) if current_date >= expected_update_date else None
+
+
+def get_ov_api20_message():
+    link = "https://docs.openvino.ai"
+    message = '[ INFO ] The model was converted to IR v11, the latest model format that corresponds to the source DL framework ' \
```

Comment:
We basically duplicate information here as we already printed out the version. And also there is no need to repeat it and there is no need to support this message in the following releases
```suggestion
    message = '[ INFO ] The model was converted to the latest model format that corresponds to the source DL framework ' \
```

Reply:
Discussed offline. Sentences with IR v11 look better and shorter. Ideally, this message should be updated regularly once IR version is changed.